src/model/element-base.ts

```diff
@@ -46,8 +47,20 @@ export abstract class ScannedElementBase implements Resolvable {
   applyHtmlComment(commentText: string|undefined) {
     if (commentText) {
       const commentJsdoc = jsdoc.parseJsdoc(commentText);
+      // Add a Warning if there are already jsdoc tags or a description for this
+      // element.
+      if (this.sourceRange &&
+          (this.description || this.jsdoc && this.jsdoc.tags.length > 0)) {
+        this.warnings.push({
+          severity: Severity.WARNING,
+          code: 'multiple-doc-comments',
+          message:
+              `More than one doc comment found for ${this.constructor.name}`,
```

Comment:
probably helpful to mention that this is probably because of one comment found in html and another found in js